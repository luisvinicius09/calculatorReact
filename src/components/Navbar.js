import React from 'react';
import { Link } from 'react-router-dom';

const Navbar = () => {
  const styles = {
    display: 'flex',
    flexDirection: 'row',
    justifyContent: 'space-between',
  };

  const stylesLink = {
    margin: '30px',
  };

  return (
    <>
      <div style={styles}>
        <div style={{ marginLeft: '10px' }}>
          <h1>Math Magicians</h1>
        </div>
        <div style={stylesLink}>
          <Link style={{ paddingRight: '10px' }} to="/">Home</Link>
          <Link style={{ paddingRight: '10px' }} to="/calculator">Calculator</Link>
          <Link to="/quote">Quote of the day</Link>
        </div>
      </div>
    </>
  );
};

export default Navbar;
