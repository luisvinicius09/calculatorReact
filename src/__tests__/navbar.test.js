import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Navbar from '../components/Navbar';

it('renders the navbar correctly', () => {
  const navbar = renderer.create(
    <BrowserRouter>
      <Navbar />
    </BrowserRouter>,
  ).toJSON();
  expect(navbar).toMatchSnapshot();
});
