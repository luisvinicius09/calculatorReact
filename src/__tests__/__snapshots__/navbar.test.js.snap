// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the navbar correctly 1`] = `
<div
  style={
    Object {
      "display": "flex",
      "flexDirection": "row",
      "justifyContent": "space-between",
    }
  }
>
  <div
    style={
      Object {
        "marginLeft": "10px",
      }
    }
  >
    <h1>
      Math Magicians
    </h1>
  </div>
  <div
    style={
      Object {
        "margin": "30px",
      }
    }
  >
    <a
      href="/"
      onClick={[Function]}
      style={
        Object {
          "paddingRight": "10px",
        }
      }
    >
      Home
    </a>
    <a
      href="/calculator"
      onClick={[Function]}
      style={
        Object {
          "paddingRight": "10px",
        }
      }
    >
      Calculator
    </a>
    <a
      href="/quote"
      onClick={[Function]}
    >
      Quote of the day
    </a>
  </div>
</div>
`;
